homework5/task1/module.c

```diff
@@ -0,0 +1,83 @@
+#include "module.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+int findMostFrequent(const char *name, int *exitCode)
+{
+    *exitCode = 0;
+    FILE* file = fopen(name, "r");
+    if (file == NULL) {
+        *exitCode = -1;
+        return 0;
+    }
+
+    int size = 0;
+    if (fscanf_s(file, "%d", &size) == NULL)
```

Comment:
```suggestion
    if (fscanf_s(file, "%d", &size) > 0)
```
fscanf_s возвращает число заполненных аргументов или EOF,  но никак не указатель